safe/definitions/post_processors.py

```diff
@@ -512,15 +512,15 @@ def post_processor_population_displacement_function(
         '"Displaced" is defined as: {displaced_concept}').format(
             displaced_concept=concepts['displaced_people']['description']),
     'input': {
-        'female_displaced':
+        'pregnant_lactating_displaced':
             {
-                'value': displaced_field,
+                'value': pregnant_lactating_displaced_count_field,
                 'type': field_input_type,
             },
         'additional_rice_ratio':
             {
                 'type': constant_input_type,
-                'value': (0.024 + 0.026) * 500 * 7 * 0.1 / 129,
+                'value': 500 * 7 * 0.1 / 129,
```

Comment:
Urf - magic numbers. Can you rather declare constants with meaningful names then create the formula from those?